New package: Hrse v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: Hrse
- Repository: https://github.com/lukebemish/Hrse.jl
- Created by: @lukebemish
- Version: v0.1.0
- Commit: 6a4db6c930471939c5d1eccbf882c0f54f3b1dbe
- Reviewed by: @lukebemish
- Reference: https://github.com/lukebemish/Hrse.jl/commit/6a4db6c930471939c5d1eccbf882c0f54f3b1dbe#commitcomment-104777324
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd636878c38d485d12cc8e23a75cf8d3c59d399814e745c24eebefa7671b6c168a2380d4ffc2785ceaead594a82e17a2fe9b4fa1d0aaa8b1e91a98a40c675e0dd6e89642aa5c5ed77d393af2d65c756c5866fc835378f0e4b5acd4f0d551c3b427d6a31a59afce16c3b4bcfe3fe3c9fb0eb9fe97d5ec9d9a359ceb14c5574273682400085bab21f779c8383fad6b2401f5f6de51e1ca7bfa90a52282d01301417c07fcd6e5099bd29b1d3eb3e0e7da686b4 -->

lukebemish commented:
I've since realized that I forgot to get my tests working on the repo, so this can probably be closed for now; I'll re-release it once those are functional. My apologies.

JuliaRegistrator committed:
New package: Hrse v0.1.0

UUID: dfa09964-7899-4f24-b5ee-10dc563fda7c
Repo: https://github.com/lukebemish/Hrse.jl.git
Tree: c5e023f481a624d27902bb46fef543e6a1530771

Registrator tree SHA: f73a20c99934db92a256057d0d83ba394036a701

goerz commented:
`HumanReadableSExpressions` or `HRSExpressions` would both seem like good package names that would be compatible with the guidelines

lukebemish commented:
Alright, thank you. If I am renaming the package (and so source repository), is there any odd behavior with Registrator I should be aware of?

goerz commented:
Not that I'm aware off… but maybe check on `#pkg-registration` on Slack

lukebemish commented:
Using JuliaRegistrator after a name change seems to create a new PR, as the package name has changed, which makes sense, so this one can now be closed
